src/containers/staking/StakePoolTable/StakePoolTableRow.test.tsx

```diff
@@ -0,0 +1,177 @@
+import '@testing-library/jest-dom/extend-expect';
+import {
+	fireEvent,
+	render,
+	screen,
+	waitForElementToBeRemoved,
+} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { ethers } from 'ethers';
+
+import {
+	MESSAGE,
+	TABLE_HEADERS,
+	TABLE_HEADERS_DISCONNECTED,
+} from './StakePoolTable.constants';
+import StakePoolTableRow, { TEST_ID } from './StakePoolTableRow';
+
+//////////
+// Mock //
+//////////
+
+var mockUserValueStaked = jest.fn();
+
+/**
+ * Note:
+ * This is directly copied from pool output in SDK
+ * as of Fri 18 Mar 2022
+ */
+const mockPoolData = {
+	instance: {
+		address: '0x9E87a268D42B0Aba399C121428fcE2c626Ea01FF',
+		userValueStaked: mockUserValueStaked,
+	},
+	content: {
+		image: '/staking/farm.png',
+		name: 'Farm WILD - WETH LP',
+		token: 'Liquidity Provider',
+		tokenTicker: 'LP',
+		domain: '/pools/lp',
+		tokenPurchaseUrl: '',
+	},
+	metrics: {
+		apy: 145.03269039595662,
+		tvl: {
+			numberOfTokens: 35576.6553275205,
+			valueOfTokensUSD: 6971564.4606713345,
+		},
+	},
+};
+
+const mockUserData = {
+	userValueUnlocked: ethers.utils.parseEther('1000'),
+	userValueLocked: ethers.utils.parseEther('1000'),
+};
+
+var mockAccountValid = { account: '1' };
+var mockAccountInvalid = { account: undefined };
+var mockAccount: { account: string | undefined } = mockAccountValid;
+
+jest.mock('@web3-react/core', () => ({
+	useWeb3React: () => mockAccount,
+}));
+
+var mockSelectStakePool = jest.fn();
+jest.mock('lib/providers/staking/PoolSelectProvider', () => ({
+	useStakingPoolSelector: () => ({
+		selectStakePool: mockSelectStakePool,
+	}),
+}));
+
+///////////
+// Setup //
+///////////
+
+const renderComponent = () => {
+	return render(
+		<table>
+			<tbody>
+				<StakePoolTableRow data={mockPoolData} rowNumber={'1'} />
+			</tbody>
+		</table>,
```

Comment:
I don't think we need `<table>, <tbody>` because it is the testing of `StakePoolTableRow ` componnent. But its okay.

Reply:
Was getting some compile errors for not putting a `<tr>` in a `<table>` and `<tbody>`, very painful